frontend/src/components/Tools/Tools.css

```diff
@@ -0,0 +1,22 @@
+.tools {
+    text-align: center;
+}
+
+.tools-header {
+    margin: 1em;
+    font-weight: 700;
+    font-size: 3em;
+}
+
+.center {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
```

Comment:
Consider putting this in App.css since others might want to use the center class as well.